src/generator/AutoRest.Go/TemplateModels/VersionTemplateModel.cs

```diff
@@ -15,12 +15,14 @@ public class VersionTemplateModel : ServiceClient
         public static readonly List<string> StandardImports = new List<string> { "fmt" };
 
         public readonly string PackageName;
+        public readonly string[] Version;
```

Comment:
minor: any reason for creating own structure for version instead of using the System.Version to provide this functionalities? [https://msdn.microsoft.com/en-us/library/system.version(v=vs.110).aspx](https://msdn.microsoft.com/en-us/library/system.version(v=vs.110).aspx)